tox.ini

```
@@ -18,6 +18,7 @@ deps =
      doctests: sphinx-copybutton
      doctests: myst-parser
      !windows-!mac: scikits.odes
+     !windows-!mac: jupyter   # For example notebooks
```

Also, shouldn't this be just `examples`? Given that `jupyter` is only used to test example notebooks? Or are we using it somewhere else too?

https://github.com/pybamm-team/PyBaMM/blob/08dc8f7b0de950e109981fd81a6bb78866548067/tox.ini#L27

```suggestion
     examples: jupyter   # For example notebooks
```

---

Yes, we are only using `jupyter` to run `examples`. 
I want to ask one thing in this regard out of curiosity that  `!windows-!mac` would already avoid specified dependency from getting installed on windows & mac, what would `examples` do?